### PR TITLE
Fix/failing unit tests

### PR DESCRIPTION
**Pull request by boecklim**
- add processor shutdown function used in unit test
- pass default policy as a parameter to arc handler

### CODE REVIEW

File: api/handler/load.go [hunk 1]
```diff
@@ -39,8 +41,16 @@ func LoadArcHandler(e *echo.Echo, logger utils.Logger) error {
 	}
 
 	var apiHandler api.HandlerInterface
+	defaultPolicySetting, found := gocore.Config().Get("defaultPolicy")
+	var defaultPolicy *bitcoin.Settings
+	if found && defaultPolicySetting != "" {
+		if err = json.Unmarshal([]byte(defaultPolicySetting), &defaultPolicy); err != nil {
```
Comment (shotasilagadzetaal) [inline api/handler/load.go:47]: Are you passing pointer to the pointer here? Is it what the function expects? Have you tested that part ? 🤔

File: metamorph/processor.go [hunk 2]
```diff
@@ -171,6 +171,22 @@ func (p *Processor) Set(req *ProcessorRequest) error {
 	return p.store.Set(spanCtx, req.Hash[:], req.StoreData)
 }
 
+// Close all channels and goroutines for graceful shutdown
+func (p *Processor) Shutdown() {
+	p.logger.Infof("Shutting down processor")
+	p.processExpiredSeenTxsTicker.Stop()
+	p.processExpiredTxsTicker.Stop()
+	if p.ch != nil {
```
Comment (shotasilagadzetaal) [inline metamorph/processor.go:179]: where do we use `p.ch` 🤔  Don't remember what channel is that. I don't see it in the code, am I missing something?

File: api/handler/load.go (hunk 1 above)
Comment (boecklim) [reply api/handler/load.go:47]: Good point! Will resolve and test.

File: metamorph/processor.go (hunk 2 above)
Comment (boecklim) [reply metamorph/processor.go:179]: Also good point. Will check that.

**Review by shotasilagadzetaal — APPROVED** [review]

**Pull request merged**